.coafile

```diff
@@ -35,7 +35,7 @@ ignore =
   _projects/README.md,
   data/locale/en/projects/README.md,
 # README.md symlinks: https://github.com/coala/coala-bears/issues/2950
-max_lines_per_file = 500
+max_lines_per_file = 1000
```

Comment:
why this?

Reply:
The lines are getting increased to around 650. And, if I combine all current commits its raised to around 800-900. That’s why I’ve increased this count to 1000. Also, my work report is failing due to this because it  too contains 650 lines.

Reply:
Can we split up app.js ?

If not, add ignores of specific files which are failing, in the appropriate PRs.

Reply:
OK! I will try to split `app.js`. If not possible, i will add an ignore

Reply:
ping - this can be removed now ?

Reply:
Also, you should do the split in the first commit , and then add your new code, 

as opposed to adding it in the first commit and then moving it in the same PR.

Reply:
Will one commit be OK? for this. As its getting harsh trying to move file content between commits 😄

Reply:
Why is it hard to create a commit which splits app.js first?

Anyways, time is short, so yes you can keep it in one commit, but you must explain what you are doing in a separate paragraph of the commit message, as the commit is contains lots of changes unrelated to the subject.

Reply:
> Why is it hard to create a commit which splits app.js first?

The first commit is indirectly adding new changes made in "projects.js"  in the "first commit" too. And, I'm trying to split but Idk why it is not happening. Its not that much difficult i know but it is getting difficult 😄 

Give a min, i will add another paragraph explaining a little-bit more about changes